AzureCalling/app/src/main/res/layout/activity_call.xml

```diff
@@ -152,4 +166,7 @@
             android:text="@string/call_hangup_cancel"
             android:gravity="center" />
     </LinearLayout>
+
+    <include android:id="@+id/list_menu_layer_for_call"
+        layout="@layout/menu_layer"/>
```

Comment:
would it be possible to specify menu, something like audio_device_selection_menu?

Reply:
Since later I plan to re-use this layer for the participants' list menu, so it better not to name it for audio device only here.